tsl/test/sql/merge_append_partially_compressed.sql.in

```diff
@@ -11,41 +11,52 @@ set enable_parallel_append to off; -- for less flaky plans
 set timescaledb.enable_decompression_sorted_merge = off;
 \set PREFIX 'EXPLAIN (analyze, costs off, timing off, summary off)'
 
-CREATE TABLE ht_metrics_compressed(time timestamptz, device int, value float);
-SELECT create_hypertable('ht_metrics_compressed','time');
-ALTER TABLE ht_metrics_compressed SET (timescaledb.compress, timescaledb.compress_segmentby='device', timescaledb.compress_orderby='time');
+CREATE TABLE mapc_test(time timestamptz, device int, value float);
+SELECT create_hypertable('mapc_test','time', create_default_indexes => false);
+ALTER TABLE mapc_test SET (timescaledb.compress, timescaledb.compress_segmentby='device', timescaledb.compress_orderby='time');
 
-INSERT INTO ht_metrics_compressed
+INSERT INTO mapc_test
 SELECT time, device, device * 0.1
-FROM generate_series('2020-01-02'::timestamptz,'2020-01-18'::timestamptz,'6 hour') time,
-generate_series(1,3) device;
+FROM generate_series('2020-01-02'::timestamptz,'2020-01-15'::timestamptz,'4 second') time,
+generate_series(1,10) device;
```

Comment:
After adding the missed ANALYZE, I had to add much more rows to keep the old plans. Also renamed it to something less generic and removed the third chunk which doesn't really add coverage.